libs/ui-watt/src/lib/components/form-field/form-field.component.scss

```diff
@@ -78,12 +85,27 @@ watt-form-field {
     display: none;
   }
 
+  [wattPrefix],
+  [wattSuffix] {
+    .watt-icon-button {
+      height: 40px;
+    }
+  }
+
   [wattPrefix] {
     margin-right: 8px;
+
+    .watt-icon-button {
+      margin-left: -12px;
+    }
   }
 
   [wattSuffix] {
     margin-left: 8px;
+
+    .watt-icon-button {
+      margin-right: -12px;
```

Comment:
question: Wouldn't it be better to move the whole component 12px to the right instead of just the button? Right now the button inside the component is moved to the right and when inspected, the component looks like this:

![image](https://user-images.githubusercontent.com/1096332/155541597-36f24589-34ac-4dfb-b851-9abb8e07ffa6.png)

Reply:
If we're targeting the whole `suffix`, icons would also be moved because it doesn't have any padding.

Reply:
What about targeting `watt-icon-button` elements with `wattSuffix` e.g. `watt-icon-button[wattSuffix]`?

Reply:
Unfortunately `watt-icon-button[wattSuffix]` won't work as this won't change the height of the button itself.

Reply:
This was more related to the `margin-right`. Instead of moving the inner button, we can move the whole `watt-icon-button` element. Wouldn't that work?

Reply:
Selector regarding margin-right and left has now been changed